__tests__/vender.spec.js

```diff
@@ -0,0 +1,99 @@
+const vendingMachine = require('./../src/vending-machine');
+
+describe('vendingMachine', () => {
+    let test;
+
+
+    beforeEach(() => {
+        test = {};
+
+        test.processData = {
```

Comment:
Make sure you aren't blindly copy/pasting the names from other projects. Names should be meaningful, and help you both read and write code.